v0.5/tools/upscale_coco/upscale_coco.py

```diff
@@ -17,7 +18,7 @@ def parse_args():
                         help='image directory')
     parser.add_argument('--annotations', '-a', type=str, default='annotations/instances_val2017.json',
                         help='annotations directory')
-    parser.add_argument('--size', default=[1400,1400], type=int, nargs='+',
+    parser.add_argument('--size', default=[1400, 1400], type=int, nargs='+',
```

Comment:
1200x1200?

Reply:
maybe we should make it a required argument instead of defaulting to something since we can also use it for the 300x300 resize.

Reply:
done, I made it required to make sure the caller knows what resolution she gets

Reply:
Sounds good!